### PR TITLE
Issue/20295 Location metadata in uploaded pictures

### PR DESCRIPTION
**Pull request by pantstamp**
This PR helps in resolving this [issue](https://github.com/wordpress-mobile/WordPress-Android/issues/20295). 

Test using WP Companion PR.

### CODE REVIEW

File: fluxc/src/main/java/org/wordpress/android/fluxc/utils/ExifUtils.kt [hunk 1]
```diff
@@ -0,0 +1,192 @@
+package org.wordpress.android.fluxc.utils
+
+import androidx.exifinterface.media.ExifInterface
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T
+import java.io.IOException
+
+object ExifUtils {
+    @Suppress("LongMethod")
```
Comment (antonis) [inline fluxc/src/main/java/org/wordpress/android/fluxc/utils/ExifUtils.kt:9]: nit pick suggestion: we may avoid the `LongMethod` warning by using a variable (`private val exifAttributes: List<String> = listOf(...`

**Review by antonis — APPROVED** [review]
Awesome work @pantstamp 🥇 
I've tested via https://github.com/wordpress-mobile/WordPress-Android/pull/20955#pullrequestreview-2107429017 and everything worked as expected for me. The code also looks good 🎉 
Left a nit pick suggestion but I'm fine proceeding with merging as is.

**Comment by AjeshRPai** [comment]
Hey Antonis 👋🏼 
Merging this PR so that we can make this into the next code freeze.

**Comment by pantstamp** [comment]
@antonis @AjeshRPai thank you for reviewing and merging this PR!

**Pull request merged**